assets/react/v3/entries/course-builder/components/additional/LiveClass.tsx

```diff
@@ -0,0 +1,290 @@
+// @TODO: will come from app config api later.
+const isPro = true;
+const hasLiveAddons = true;
+
+const LiveClass = () => {
+  const [showMeetingForm, setShowMeetingForm] = useState<MeetingType | null>(null);
+  // @TODO: will come from app config api later.
+  const [meetings, setMeetings] = useState<Meeting[]>([]);
+
+  const zoomMeetings = meetings.filter(meeting => meeting.type === 'zoom');
+  const googleMeetMeetings = meetings.filter(meeting => meeting.type === 'google_meet');
+
+  const zoomBtnRef = useRef<HTMLButtonElement>(null);
+  const googleMeetBtnRef = useRef<HTMLButtonElement>(null);
+  const jitsiBtnRef = useRef<HTMLButtonElement>(null);
+
+  return (
+    <div css={styles.liveClass}>
+      <span css={styles.label}>
+        {__('Live Class', 'tutor')}
+        {!isPro && <SVGIcon name="crown" width={24} height={24} />}
+      </span>
+      <Show
+        when={isPro}
+        fallback={
+          <EmptyState
+            size="small"
+            removeBorder={false}
+            emptyStateImage="https://via.placeholder.com/360x360"
+            emptyStateImage2x="https://via.placeholder.com/760x760"
+            imageAltText="No live class addons found"
+            title={__('Make the learning more interactive and fun using Live class feature! ', 'tutor')}
+            description={__(
+              'when an unknown printer took a galley of type and scrambled it to make a type specimen book.',
+              'tutor'
+            )}
+            actions={
+              <Button
+                icon={<SVGIcon name="crown" width={24} height={24} />}
+                onClick={() => {
+                  alert('@TODO: Will be implemented in future');
+                }}
+              >
+                {__('Get Tutor LMS Pro', 'tutor')}
+              </Button>
+            }
+          />
+        }
+      >
+        <Show
+          when={hasLiveAddons}
+          fallback={
+            <EmptyState
+              size="small"
+              removeBorder={false}
+              emptyStateImage="https://via.placeholder.com/360x360"
+              emptyStateImage2x="https://via.placeholder.com/760x760"
+              imageAltText="No live class addons found"
+              title={__('You can use this feature by activating Google meet, Zoom or Jitsi from addons', 'tutor')}
+              description={__(
+                'when an unknown printer took a galley of type and scrambled it to make a type specimen book.',
+                'tutor'
+              )}
+              actions={
+                <Button
+                  variant="secondary"
+                  onClick={() => {
+                    alert('@TODO: Will be implemented in future');
+                  }}
+                >
+                  {__('Go to addons', 'tutor')}
+                </Button>
+              }
+            />
+          }
+        >
+          <div
+            css={styles.meetingsWrapper({
+              isMultiple: zoomMeetings.length > 0,
```

Comment:
Could you please explain this? why are we deciding is multiple if the zoomMeetings length is greater than 0?

Reply:
It is for displaying borders when there are any meetings. I should not have named it this way. It can be showBorder.